geth/api/api_test.go

```diff
@@ -123,5 +123,5 @@ func (s *APITestSuite) TestRaceConditions() {
 	}
 
 	time.Sleep(2 * time.Second) // so that we see some logs
-	s.api.StopNode()            // just in case we have a node running
+	_ = s.api.StopNode()        // just in case we have a node running
```

Comment:
Well, this is a test and not of `s.api.StopNode` and we don't really care.

Reply:
it's only fixing errcheck linter warning.

Reply:
`//nolint: errcheck`?

Reply:
It's another way, but i've changed it to nolint. It's more faster way to find all nolint tags.